src/main/java/core/basesyntax/servise/operation/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.servise.operation;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void apply(String fruit, int quantity, Map<String, Integer> storage) {
```

Comment:
According to the checklist, handling the 'Balance' operation should also check the result balance before saving it in the Storage - it should be positive. Consider adding a check to ensure that the quantity is not negative before updating the storage.